MapleServer2/PacketHandlers/Game/RequestItemBreakHandler.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Maple2Storage.Types;
+using Maple2Storage.Types.Metadata;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Data.Static;
+using MapleServer2.Packets;
+using MapleServer2.Servers.Game;
+using MapleServer2.Tools;
+using MapleServer2.Types;
+using Microsoft.Extensions.Logging;
+
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class RequestItemBreakHandler : GamePacketHandler
+    {
+        public override RecvOp OpCode => RecvOp.REQUEST_ITEM_BREAK;
+
+        public RequestItemBreakHandler(ILogger<RequestItemBreakHandler> logger) : base(logger) { }
+
+        private Dictionary<short, long> DismantleSlots;
+        private Dictionary<int, int> Rewards;
```

Comment:
While I'm not sure exactly where else this should live, we shouldn't be persisting state in handlers. Feel free to start a convo in Discord or move it to another file (you can also add the helper methods that you are using here)